modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,69 @@
+= API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API Groups enables you to bundle your APIs and specific resources to solve specific user needs. Instead of individually using a list of resources and APIs, the API users can request access to and use these APIs as a package that solves a specific problem for them. 
+
+When you create an API Group, you can create different instances of that group with different SLAs and rate limit conditions based on the problem it must solve. You can then publish the API Group to Anypoint Exchange for users to be able to subscribe to use the package.
```

Comment:
```suggestion
When you create an API Group, you can customize instances of that group with different SLAs and rate limit conditions based on the problem the instance must solve. You can then publish the API Group to Anypoint Exchange so that users can subscribe to the package.
```

Reply:
Done! Thanks!